diff --git a/src/main/osd/osd.c b/src/main/osd/osd.c
index e9a430fbc..05776a2eb 100644
--- a/src/main/osd/osd.c
+++ b/src/main/osd/osd.c
@@ -613,7 +613,7 @@ static void osdShowStats(uint16_t endBatteryVoltage)
 static void osdShowArmed(void)
 {
     displayClearScreen(osdDisplayPort);
-    displayWrite(osdDisplayPort, 12, 7, "ARMED");
+    displayWrite(osdDisplayPort, 9, 7, "AKRAMODE");
 }
 
 STATIC_UNIT_TESTED void osdRefresh(timeUs_t currentTimeUs)
